src/ContentUtils.elm

```diff
@@ -5,6 +5,7 @@ import List
 import Pages
 import Date.Extra
 import Posts
+import String exposing (..)
```

Comment:
For modules like this I wouldn't expose any methods. So I'd `import String` and then use `String.contains`. I like having the functions qualified with the module they come from.